### DISCUSSION TITLE
LCD (ST7920) & OLED (SSD1322) Displays (4 Wire SPI) compatible with this library?

### DISCUSSION THREAD

**Discussion by NWSpitfire** [forntoh/LcdMenu#176] [General]
Hi!

I have come across this library on my quest to implement a simple menu to display on my 128x64 SPI LCD (ST7920) and 256x64 SPI OLED (SSD1322). 

It looks great, but I see it is mainly geared for character displays like 1602 and 2004 types. Will it work with my dot matrix displays?

Also, is there a way to implement hot-keys to auto-select sub menus. For example, if I had a sub-menu for Electrical systems, could I press a hot-key and navigate straight to it from anywhere?

Thank you!

**Comment by forntoh** [comment]
Hi @NWSpitfire, this is a WIP, very soon anyone will be able to create an adapter for any display type which they use.
See #153

Once it's released, you can explore creating an adapter which will handle drawing the menu for LCD (ST7920) & OLED (SSD1322).
